tsconfig.json

```diff
@@ -10,11 +10,8 @@
     "noUnusedParameters": true,
     "sourceMap": true,
     "strict": true,
-    "noImplicitReturns": true,
```

Comment:
We better leave this enabled, as it is not enabled by default: https://www.typescriptlang.org/tsconfig#noImplicitReturns

Reply:
Ok!